File: sbin/webhook_info.php
#!/usr/bin/env php
<?php
  require_once __DIR__.'/../core/common.php';

  //$Config   = new U2\Config( ROOT.'/config.json' );
  //$Telegram = new U2\Telegram( $Config );

  $Response = \Tg\Core\Webhook::info();
  
  tgdump( $Response ) . PHP_EOL;  
?>

File: bundles/php/cli/webhook_info.go
#!/usr/bin/env php
<?php
namespace Tg\Core;
use \Tg\Exception\Fatal;

require_once __DIR__.'/../autoloader.php';
try{
  $Response = Webhook::info();
  tgdump( $Response ).PHP_EOL;
}
catch(\Throwable $e){
  new Fatal($e->getMessage());
}
?>
